Makefile

```diff
@@ -142,7 +144,12 @@ GOARCH=$(lastword $(subst -, ,$*))
 build/mgmt-%: $(GO_FILES) | bindata lang
 	@echo "Building: $(PROGRAM), os/arch: $*, version: $(SVERSION)..."
 	@# reassigning GOOS and GOARCH to make build command copy/pastable
+ifneq ($(NEWGOLANG),)
```

Comment:
Can we inline this with a shell `if` like in the old version of the commit instead? This will keep it slightly faster when we run `make <TAB>`

Reply:
Sure no prob.

Reply:
Hmm. Not working as expected. I'll get back to it soon.